pkg/sinit/init.go

```diff
@@ -245,6 +261,14 @@ func InitializeLogging(ctx context.Context, config LoggingConfig) error {
 			strc.SetLogger(logger)
 		}
 
+		// configure standard library logger
+		if config.StdlibLogConfig.Enabled {
+			logger := slog.NewLogLogger(handlerMulti, slog.LevelDebug)
+			writer := &logLoggerWriter{dest: logger}
+			log.SetFlags(config.StdlibLogConfig.Flags)
+			log.SetOutput(writer)
+		}
```

Comment:
This is the gist of the patch. To explain, standard library contains `log` and `log/slog` packages and also a type and a helper function `NewLogLogger` that works as a proxy. The default `log` logger cannot be swapped with a new one therefore a writer interface must be implemented to be able to call `SetOutput`.

But this is also quite useful because then a writer that writes to `log/slog` can be easily obtained by calling `log.Writer()` and this is exactly the point: this will be used in `composer` to dump configuration values during start.

Reply:
I wonder if it might be worth putting an (abbreviated) version of this comment into the code as a comment? It could be as simple as `// configure standard library logger` -> `// configure standard library logger to forward to  log/slog when enabled` (but maybe a tiny bit more, idy)

Reply:
It seems this was not addressed?